fonts/fonts.css

```diff
@@ -1,8 +1,15 @@
 @font-face {
   font-family: 'Montserrat';
-  src:
-    url('montserrat-600-normal.woff2') format('woff2'),
-    url('montserrat-600-normal.woff') format('woff');
-  font-weight: normal;
+  src: url('montserrat-semibold.woff2') format('woff2'),
+       url('montserrat-semibold.woff') format('woff');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/max_osipov/pullRequest?prid=5654195)